rand_hc/src/hc128.rs

```diff
@@ -103,8 +100,13 @@ impl SeedableRng for Hc128Rng {
     }
 
     #[inline]
-    fn from_rng<R: RngCore>(rng: R) -> Result<Self, Error> {
-        BlockRng::<Hc128Core>::from_rng(rng).map(Hc128Rng)
+    fn from_rng(rng: impl RngCore) -> Self {
+        Hc128Rng(BlockRng::<Hc128Core>::from_rng(rng))
+    }
+
+    #[inline]
+    fn try_from_rng<R: TryRngCore>(rng: R) -> Result<Self, R::Error> {
+        BlockRng::<Hc128Core>::try_from_rng(rng).map(Hc128Rng)
     }
```

Comment:
In many cases, I think we need to implement both of these. Annoying, but not terrible.